concent-builder/deploy.yml

```diff
@@ -0,0 +1,36 @@
+- hosts:
+    - concent-builder
+  vars_files:
+    - consts.yml
+  tasks:
+    - become:      yes
+      remote_user: builder
+      block:
+        - name:    Google Cloud Engine service account activating
+          command: gcloud auth activate-service-account concent-builder-service@devel-concent-golem-network.iam.gserviceaccount.com --key-file "/home/{{ shared_user }}/build/concent-secrets/devel-concent-golem-network-198b161e7c8a.json"
+
+        - name:    Google Cloud Engine service prepare project
+          command: gcloud container clusters get-credentials concent-dev --zone europe-west3-a --project devel-concent-golem-network
+
+        - name:    Changing perm of push-docker-images.sh
+          file:
+            path:  "{{ build_dir }}/concent-deployment/kubernetes/build/push-docker-images.sh"
+            mode:  0700
```

Comment:
1) Why `0700`? And there's no reason not to give other users permissions to read and executed the file. Use `0755`.

2) Changing these permissions should be part of `build.yml`. Maybe we should even do this in our Makefile.

3) Actually changing them should not be necessary at all. Our Makefile even has specific instructions that preserve the permissions. Ansible can lose them when uploading times (it's a long-standing bug) but you're not uploading these files with Ansible. You're getting them directly from the repository. This step is not necessary. Remove it and fix the underlaying problem.